packages/mdc-list/README.md

```diff
@@ -384,3 +384,15 @@ And the basic CSS is relatively trivial
   align-items: flex-end;
 }
 ```
+
+### Sass Mixins
+
+Mixin | Description
+--- | ---
+`mdc-list-item-primary-text-ink-color($color)` | Sets the ink color of the primary text
+`mdc-list-item-secondary-text-ink-color($color)` | Sets the ink color of the secondary text
+`mdc-list-item-icon-fill-color($color)` | Sets the fill color of the icon (start detail)
+`mdc-list-item-icon-ink-color($color)` | Sets the ink color of the icon (start detail)
+`mdc-list-item-meta-ink-color($color)` | Sets the ink color of the metadata (end detail)
```

Comment:
I generally want to migrate the names of things to match the spec, cuz it can get really confusing when the names don't match. I was planning to do the actual renaming part in a separate PR, but it felt like it would still make sense to have the documentation match the spec.

The spec calls the first/leading the "Supporting visuals tile", which is super ugly for several reasons, so I didn't use any variations of that phrase.

I figured that because the mixins are only used for icons (they are meaningless when an image avatar is used), having `icon` in the name made more sense than `start-detail` or `leading-tile` or something. But it does have the unfortunate side effect of suggesting that it will change the color of _all_ icons, including trailing icons, which is not actually the case.

So I'm open to suggestions for naming, but I want to try to follow the spec if possible. If the names used in the spec make that difficult or awkward, we can always talk to the designers/writers about updating the spec with better, shorter names for elements.

Reply:
Renamed "icon" to "graphic" after talking to our designer. The spec will be updated to call it a "supporting graphic" instead of "supporting visual".